src/Appium.Net/Appium/Service/AppiumLocalService.cs

```diff
@@ -184,17 +184,17 @@ private bool Ping(TimeSpan span)
             Uri service = ServiceUrl;
             if (service.IsLoopback || IP.ToString().Equals(AppiumServiceConstants.DefaultLocalIPAddress))
             {
-                status = new Uri("http://localhost:" + Convert.ToString(Port) + "/wd/hub/status");
+                status = new Uri("http://localhost:" + Convert.ToString(Port) + "/status");
```

Comment:
`localhost` might be pointing to an ipv6 address by default, so the status check won't work if the server is listening on ipv4 one. I would rather be quite careful while hardcoding it

Reply:
I assume this is something that wasn't handled by now anyway, or it's something new introduced with appium 2?

Reply:
it's unrelated to the server. The issue is a pure client one.
I do not know how it was implemented/working before, just pointing to a potential issue

Reply:
Not sure I understand the potential issue. 
When I send a request with the below request which includes the status check, the response returns OK. 
And the HostNameType is IPv4 with localhost.
![image](https://user-images.githubusercontent.com/59066376/197347946-afa95db4-f2cf-42e3-a8e4-64406e3e7ec7.png)

![image](https://user-images.githubusercontent.com/59066376/197347951-6a6f47c1-b088-41a9-a735-8352f99084ea.png)